pkg/scheduler/api/queue_info.go

```diff
@@ -56,3 +56,20 @@ func (q *QueueInfo) Clone() *QueueInfo {
 		Queue:  q.Queue,
 	}
 }
+
+// ReclaimAble return whether queue is reclaimAble
+func (q *QueueInfo) ReclaimAble() bool {
```

Comment:
It's better to set default value in admission controller; so user can also check this value in cli :)

Reply:
Since there is no queue admission now, this may require another commit to handle queue validating and mutating.

Reply:
yes, that's ok to open another PR for that :)